### PR TITLE
Fix tool run uses ellipses slightly too soon

### LINKED ISSUE #7984: Tool run uses ellipses slightly too soon
With 4 items probably makes sense to print all four since there's room for 3 + ellipsis? And only if ≥5 then use ellipses

image

### PR DESCRIPTION
**Pull request by WilliamHolden**
fixes #7984

### CODE REVIEW

File: client/galaxy/scripts/mvc/tool/tool-form.js [hunk 1]
```diff
@@ -68,15 +68,15 @@ var View = Backbone.View.extend({
                                     );
                                 } else {
                                     Galaxy.modal &&
```
Comment (dannon) [inline client/galaxy/scripts/mvc/tool/tool-form.js]: @guerler Do you remember why we needed to have this guard in here?  Is it irrelevant now, and safe to drop?

**Comment by martenson** [comment]
switching to WIPdue to https://github.com/galaxyproject/galaxy/issues/7984#issuecomment-499227590

File: client/galaxy/scripts/mvc/tool/tool-form.js (hunk 1 above)
Comment (guerler) [reply client/galaxy/scripts/mvc/tool/tool-form.js]: Yes, it should be safe to remove that safe guard now.

**Comment by martenson** [comment]
Galaxy

**Review by dannon — APPROVED** [review]

**Pull request merged**